context/context.go

```diff
@@ -78,8 +82,18 @@ func RetrieveDnnInformation(Snssai models.Snssai, dnn string) *SnssaiSmfDnnInfo
 }
 
 func AllocateLocalSEID() uint64 {
+	if smfContext.LocalSEIDCount >= 5000 {
```

Comment:
We wish to go beyond this. and our resource sharing code should be able to address this limitation